server/utils/middleware.js

```diff
@@ -28,3 +28,21 @@ export function ifNoUser401(req, res, next) {
   }
   return res.status(401).end();
 }
+
+export function sendUserNotice(message, type = 'info') {
+  return function(req, res, next) {
+    const user = req.user;
+    if (!user) {
+      return next();
+    }
+    const email = req.user.email;
+    if (!email) {
+      console.log('sending user notice');
+      req.flash(type, { msg:
+        message ||
+         'You have not updated your email! Please update it in Settings'
```

Comment:
We should only deploy this once we have email verification functionality in place. We only want to pester campers once. The text should read something like:
"Please confirm your email address when you get a moment in <a href='/settings'>your settings</a>"

Note that we shouldn't use exclamation points, as they can be interpreted as commands, which are kind of rude.